src/app/reflection/reflection.page.ts

```diff
@@ -0,0 +1,22 @@
+import {  Component, OnInit } from '@angular/core';
+import { Location } from '@angular/common';
+
+@Component({
+  selector: 'app-reflection',
+  templateUrl: './reflection.page.html',
+  styleUrls: ['./reflection.page.scss']
+})
+export class ReflectionPage  implements OnInit {
+  headerConfig = {
+    showBackButton:false,
+  };
+  reflectionPointsList = ["REFLECTION_POINT_ONE", "REFLECTION_POINT_TWO", "REFLECTION_POINT_THREE"]
```

Comment:
will it always be three items?

Reply:
Sure can check this and confirm. However this page is static as of now and its functionality integration will be done in the next drop. So by that time will clarify this